src/pep8ext_naming.py

```diff
@@ -436,7 +436,4 @@ def _extract_names(assignment_target):
 
 
 def is_mixed_case(name):
-    stripped_name = name.strip('_')
-    head = stripped_name[:1]
-    tail = stripped_name[1:]
-    return head.islower() and tail and not (tail.islower() or tail.isdigit())
+    return name.lstrip('_')[:1].islower() and name.lower() != name
```

Comment:
This test looks good. If we're after performance, I think it might be slightly better to reverse the checks:

```python
return name.lower() != name and name.lstrip('_')[:1].islower()
```

That first condition should cover the majority of the cases. The second condition does more work and creates more intermediate objects, so it's better to defer it.

Reply:
Done.